Issue #23 Our own text generator

RomanZorkin opened the pull request:
Closes #23 
In this case I have one linter mistake in "MakeWord" class - "random_word" method. Linter suspects random generator of a security breach:

"S311 Standard pseudo-random generators are not suitable for security/cryptographic purposes. "

It seems to me that the "random_word" function has no binding to security.

I also need to supplement the text files with a large number of words

RomanZorkin committed:
Issue #23 Our own text generator

Closes #23
In this case I have one linter mistake in "MakeWord" class - "random_word" method. Linter suspects random generator of a security breach:

"S311 Standard pseudo-random generators are not suitable for security/cryptographic purposes. "

It seems to me that the "random_word" function has no binding to security.

I also need to supplement the text files with a large number of words

RomanZorkin commented on geopatterns_demo/random_phrase.py:
```diff
@@ -1,6 +1,22 @@
+        """Find random word in text file."""
+        with open(self.text_file, 'r') as text:
+            lines = text.readlines()
+            return random.choice(lines).rstrip('\n')
```
There is linter problem:

 S311 Standard pseudo-random generators are not suitable for security/cryptographic purposes. [flake8-bandit]
  return random.choice(lines).rstrip('\n')

anatoly-scherbakov replied:
`bandit` has to complain about this but we're not using this for any cryptographic or security purposes, so I guess `random.choice()` is pretty much okay. We can ignore this.

It is also a good practice to add a comment to explain why the `noqa` was added.

```suggestion
            # random.choice() is considered to be insecure by flake8-bandit, but we're not using this for any kind of cryptographic or security related purpose.
            return random.choice(lines).rstrip('\n')   # noqa: S311
```

anatoly-scherbakov commented on geopatterns_demo/random_phrase.py:
```diff
@@ -1,6 +1,22 @@
-    """Method create random phrase using loremipsum library."""
-    return loremipsum.generate(1, 'short')
+    """Glues  and return a phrase from random words."""
+    words_list = [MakeWord(word.value).random_word() for word in Thesaurus]
```
```suggestion
    words_list = [MakeWord(file_path.value).random_word() for file_path in Thesaurus]
```

The variable should reflect its name, so perhaps `file_path` would be a more appropriate name.

anatoly-scherbakov commented on geopatterns_demo/models.py:
```diff
@@ -35,3 +35,10 @@ class PhraseDescription(BaseModel):
+class Thesaurus(Enum):
+    """List of dictionaries with words."""
+
+    ADJECTIVES = 'geopatterns_demo/dictionary/adjectives.txt'
```
```suggestion
    ADJECTIVES = Path(__file__).parent / 'dictionary/adjectives.txt'
```

Check out `Path` class from `pathlib` library. It helps working on file paths A LOT.

RomanZorkin replied:
Whether there is a rule for the correct order of classes. Let's say alphabetically?

RomanZorkin committed:
Merge branch 'anatoly-scherbakov:master' into Issue#23_ourselves_random_phrase

RomanZorkin committed:
Issue #23 Our own text generator

In module random_phrase.py
1. added comment  about the exception in linter test
2. fixed the variable name in the make_random_text function

RomanZorkin committed:
Issue #23 Our own text generator

In module random_phrase.py
1. added comment  about the exception in linter test
2. fixed the variable name in the make_random_text function

RomanZorkin committed:
Issue #23 Our own text generator

In module models.py used pathlib library to path administarte
